sonar-html-plugin/src/main/resources/org/sonar/l10n/web/rules/Web/ImgWithoutAltCheck.html

```diff
@@ -1,22 +1,76 @@
 <p>The <code>alt</code> attribute provides a textual alternative to an image.</p>
-<p>It is used whenever the actual image cannot be rendered.</p>
-<p>Common reasons for that include:</p>
-<ul>
-  <li> The image can no longer be found </li>
-  <li> Visually impaired users using a screen reader software </li>
-  <li> Images loading is disabled, to reduce data consumption on mobile phones </li>
-</ul>
-<p>Empty <code>alt</code> attributes are not allowed, since purely decorative images should be specified using CSS, not using the <code>img</code>
-tag.</p>
+<p> It is used whenever the actual image cannot be rendered.</p>
+<p> Common reasons for that include:</p>
+<p> * The image can no longer be found</p>
+<p> * Visually impaired users using a screen reader software</p>
+<p> * Images loading is disabled, to reduce data consumption on mobile phones</p>
```

Comment:
There was an issue with rule-api. The lines above should be `<ul> <li> ....` instead of a list of `<p>` elements.
Please rerun rule-api, I updated Jira ticket description so that it will be correctly generated now